src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -219,7 +219,24 @@ <h6 class="alert alert-warning no-icon additional" ng-show="!queryIsRunning && !
                     <h6>{{'common.error' | translate}}<span ng-show="yasr.results.getException().status > 0"> {{yasr.results.getException().status}}: {{yasr.results.getException().statusText}}</span></h6>
                 </div>
 
-                <div class="plaintext">{{yasr.results.getException().responseText}}</div>
+                <div ng-if="!fullErrorMessage && sparqlQueryErrorInfo && sparqlQueryErrorInfo.shortenErrorMessage">
+                    <div class="show-full-message">
```

Comment:
There are too much containers around these links here. Can we simplify this?

Reply:
Also, this `show-full-message` class sounds like something which should be put on some actionable component like button or link, but here it is set on a div tag.

Reply:
refactored